pkg/controller/serviceaccount/serviceaccounts_controller.go

```diff
@@ -185,7 +185,7 @@ func (c *ServiceAccountsController) processNextWorkItem() bool {
 func (c *ServiceAccountsController) syncNamespace(key string) error {
 	startTime := time.Now()
 	defer func() {
-		klog.V(4).Infof("Finished syncing namespace %q (%v)", key, time.Since(startTime))
+		klog.V(4).InfoS("Finished syncing namespace", "namespace", key, "startTime", time.Since(startTime))
```

Comment:
Maybe it’s the duration instead of the start time?

Reply:
done